config/application.yml.default

```diff
@@ -312,7 +312,7 @@ development:
   hmac_fingerprinter_key: a2c813d4dca919340866ba58063e4072adc459b767a74cf2666d5c1eef3861db26708e7437abde1755eb24f4034386b0fea1850a1cb7e56bff8fae3cc6ade96c
   hmac_fingerprinter_key_queue: '["11111111111111111111111111111111", "22222222222222222222222222222222"]'
   identity_pki_local_dev: true
-  idv_api_enabled_steps: '["password_confirm", "personal_key","personal_key_confirm"]'
+  idv_api_enabled_steps: '[]'
```

Comment:
😅 I've actually been thinking this is a change we should make, but did you intend to include it here?

Reply:
No, thanks for pointing this out. Will revert

Reply:
Reverted in [5436c45](https://github.com/18F/identity-idp/pull/6629/commits/5436c451e41c38743dce4d9d5b747bfc3e4cfba7)